handbook/engineering/hiring/engineering-manager-web.md

```diff
@@ -33,7 +35,7 @@ Learn more about what it is like to work at Sourcegraph by reading [our handbook
     - 45m [Loïc Guychard](../../../company/team/index.md#loïc-guychard) (Previous web team eng manager)
 1. Interviews.
    - 1h [Team collaboration](https://github.com/sourcegraph/interviews/blob/master/engineering/team-collaboration.md).
```

Comment:
Can you update this with short description too?

Reply:
added in another pull request.